doc/concepts/replication/repl_architecture.rst

```diff
@@ -8,62 +8,95 @@ Replication architecture
+
+*   Data definition operations on **temporary spaces** (:doc:`created </reference/reference_lua/box_schema/space_create>` with ``temporary = true``), such as creating/dropping, adding indexes, and truncating, are written to the WAL, since information about temporary spaces is stored in non-temporary system spaces, such as :ref:`box.space._space <box_space-space>`.
+
+*   Data change operations on temporary spaces are not written to the WAL and are not replicated.
 
 ..  _replication-local:
 
-Data change operations on **replication-local** spaces
-(spaces :doc:`created </reference/reference_lua/box_schema/space_create>`
-with ``is_local = true``)
-are written to the WAL but are not replicated.
-
-To create a valid initial state, to which WAL changes can be applied, every
-instance of a replica set requires a start set of
-:ref:`checkpoint files <index-box_persistence>`, such as .snap files for memtx
-and .run files for vinyl. A replica joining an existing replica set, chooses an
-existing master and automatically downloads the initial state from it. This is
-called an **initial join**.
-
-When an entire replica set is bootstrapped for the first time, there is no
-master which could provide the initial checkpoint. In such a case, replicas
-connect to each other and elect a master, which then creates the starting set of
-checkpoint files, and distributes it to all the other replicas. This is called
-an **automatic bootstrap** of a replica set.
-
-When a replica contacts a master (there can be many masters) for the first time,
-it becomes part of a replica set. On subsequent occasions, it should always
-contact a master in the same replica set. Once connected to the master, the
-replica requests all changes that happened after the latest local LSN (there
-can be many LSNs -- each master has its own LSN).
-
-Each replica set is identified by a globally unique identifier, called the
-**replica set UUID**. The identifier is created by the master which creates the
-very first checkpoint, and is part of the checkpoint file. It is stored in
-system space :ref:`box.space._schema <box_space-schema>`. For example:
+*    Data change operations on **replication-local** spaces (:doc:`created </reference/reference_lua/box_schema/space_create>` with ``is_local = true``) are written to the WAL but are not replicated.
+
+
+To learn how to enable replication, check the :ref:`Bootstrapping a replica set <replication-setup>` guide.
+
+
+.. _replication_stages:
+
+Replication stages
+~~~~~~~~~~~~~~~~~~
+
+To create a valid initial state, to which WAL changes can be applied, every instance of a replica set requires a start set of :ref:`checkpoint files <index-box_persistence>`, such as ``.snap`` files for memtx and ``.run`` files for vinyl.
+A replica uses the fiber called :ref:`applier <memtx-replication>` to receive the changes from a remote node and apply them to the replica's arena.
+The applier goes through the following stages:
```

Comment:
This looks mixed up a little.
These are not 'applier stages'. You mention all the applier stages below, in `box.info.replication` section.

These look rather like "replication stages". In general, of a whole cluster. Or maybe of a single instance.
And if we're talking of replication stages and not applier states, I don't think we should mention "connect" or "auth". I'd go with the following:

"bootstrap" - when noone has the initial snapshot to boot from, and they all choose a bootstrap leader, which will create the snapshot by itself so that everyone else can join to this leader.
"join" - when some particular replica downloads the master's snapshot and the master registers it in _cluster space. 
"follow" - when the replicas replicate WALs from the master normally

Reply:
Thanks for clarifying, will update. I tried to adapt information from the corresponding Medium [blog post](https://medium.com/@tarantool/replication-from-scratch-in-five-simple-steps-is-impossible-1ef046b08216) :)

```
Replication status is easier to monitor from a replica. 
Its applier successively goes through the following states: CONNECT, AUTH, JOIN (INITIAL + FINAL), SUBSCRIBE (SYNC + FOLLOW).
```

So, is it better to not mention the `SUBSCRIBE` stage at all?

Reply:
"follow" and "subscribe" are synonyms. I just find "follow" more understandable, because that's what's shown in `box.info.replication[id].upstream.status` and because we use 'follow' in informal speech.